Sources/Pipelines/CodeGen/SourceCodeFolderDistributorStage.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  SourceCodeFolderDistributorStage.swift
+//  
+//
+//  Created by volodina on 14.02.2022.
+//
+
+import Foundation
+import CodeGenerator
+import Common
+
+/// Fixes destinationPath: add nested folder name according specification in order to generate files in separate packages
+public struct SourceCodeFolderDistributorStage : PipelineStage {
```

Comment:
So, to be honest, from this description i got nothing (:
May be example? Reference to documentation or something